community/cypher/cypher-compiler-2.2/src/test/scala/org/neo4j/cypher/internal/compiler/v2_2/planner/LogicalPlanningTestSupport2.scala

```diff
@@ -257,13 +145,14 @@ trait LogicalPlanningTestSupport2 extends CypherTestSupport with AstConstruction
           tokenResolver.resolve(ast)(newTable, planContext)
           val unionQuery = ast.asUnionQuery
           val metrics = metricsFactory.newMetrics(planContext.statistics, newTable)
-          val context = LogicalPlanningContext(planContext, metrics, newTable, queryGraphSolver, QueryGraphCardinalityInput(Map.empty, Cardinality(1)))
+          val context = LogicalPlanningContext(planContext, metrics, newTable, queryGraphSolver, QueryGraphCardinalityInput.empty)
           val plannerQuery = unionQuery.queries.head
           val resultPlan = strategy.internalPlan(plannerQuery)(context)
           SemanticPlan(resultPlan.endoRewrite(unnestApply), newTable)
       }
     }
 
+    // TODO: Remove
```

Comment:
Strange must be a refactoring artifact. I don't remember adding this, will remove it.